web_tool_script_1.R

```diff
@@ -111,6 +111,21 @@ portfolio <- process_raw_portfolio(
   grouping_variables
 )
 
+# information of coverage and coverage loses for all funds in raw_portfolio
+fund_coverage <- get_fund_coverage(
+  portfolio_raw,
+  fin_data,
+  fund_data,
+  currencies,
+  grouping_variables
+)
+
+# reduce information on fund coverage to a data frame that can be shared with users
+fund_coverage_summary <- summarize_fund_coverage(fund_coverage)
+
+# list ISINs of unknown funds in funds. the list includes value_usd to estimate importance of the isin for o
+unknown_funds_in_funds <- list_unknown_funds_in_funds(portfolio)
```

Comment:
is this meant to be the portfolio or the total_portfolio or portfolio_total as an input?

Reply:
portfolio.. is right here.

Reply:
as the relevant dataframe during the run of web_tool_script_1 is "portfolio".. BUT! in the function definition in 0_portfolio_... i used portfolio_total as variable name, for debugging reasons, because the return of 0_portfolio_... i used portfolio_total is portfolio_total

Reply:
Great